source/reference/minio-mc/mc-batch-describe.rst

```diff
@@ -0,0 +1,103 @@
+.. _minio-mc-batch-describe:
+
+=====================
+``mc batch describe``
+=====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc batch describe
+
+.. versionchanged:: MinIO RELEASE.2022-10-08T20-11-00Z or later
+
+Syntax
+------
+
+.. start-mc-batch-describe-desc
+
+The :mc:`mc batch describe` command outputs the job definition for a specified job ID.
+
+.. end-mc-batch-describe-desc
+
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following command outputs a list of all jobs currently in progress on the ``myminio`` alias.
```

Comment:
This sounds different - assuming that it's a "list or describe" if you omit the job id?

Reply:
This is a copy/paste fail.

Reply:
Can't believe I just left so much unedited in this file. But cleaned up now.